include/rocksdb/utilities/options_type.h

```diff
@@ -306,6 +306,52 @@ class OptionTypeInfo {
         });
   }  // End OptionTypeInfo::Enum
 
+  template <typename T>
+  static OptionTypeInfo Enum(
```

Comment:
Would be good to share code with the previous definition of ```Enum()```, since the only addition here is the ```flags``` argument.